netplug uses nested functions, and gcc generates a warning that
is turned into an error:

cc1: warnings being treated as errors
if_info.c: In function 'ifsm_scriptdone':
if_info.c:289: error: generating trampoline in object (requires executable stack)

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: netplug-1.2.9/Makefile
===================================================================
--- netplug-1.2.9.orig/Makefile	2010-05-09 21:07:40.000000000 +0200
+++ netplug-1.2.9/Makefile	2010-05-09 21:09:32.000000000 +0200
@@ -9,7 +9,7 @@
 
 install_opts :=
 
-CFLAGS += -Wall -Werror -std=gnu99 -DNP_ETC_DIR='"$(etcdir)"' \
+CFLAGS += -Wall -std=gnu99 -DNP_ETC_DIR='"$(etcdir)"' \
 	-DNP_SCRIPT_DIR='"$(scriptdir)"' -ggdb3 -O3 -DNP_VERSION='"$(version)"'
 
 netplugd: config.o netlink.o lib.o if_info.o main.o
